modules/installation-configuration-parameters.adoc

```diff
@@ -1505,6 +1513,14 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 |The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type] for control plane machines.
 |Control plane machines must use the `pd-ssd` disk type, which is the default.
 
+|`controlPlane.platform.gcp.osImage.project`
+|Optional. By default, the installation program downloads and installs the {op-system-first} image that is used to boot control plane machines. You can override the default behavior by specifying the location of a custom {op-system} image that the installation program is to use for control plane machines only.
+|String. The name of GCP project where the image is located.
+
+|`controlPlane.platform.gcp.osImage.name`
```

Comment:
Same here about this one coming before the above one, if it's alphabetical

And also same wording suggestions as the two earlier in the file.

Same for the below two fields as well, if you can check all